### PR TITLE
Add tests for TransformManager

### PR DESCRIPTION
**Pull request by chcn**

### CODE REVIEW

File: table/server/master/src/test/java/alluxio/master/table/transform/TransformManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+    checkTransformJobInfo(mTableMaster.getTransformJobInfo(jobId2), TABLE2, DEFINITION2, jobId2,
+        Status.FAILED, "error");
+    checkTransformJobInfo(mTableMaster.getTransformJobInfo(jobId3), TABLE3, DEFINITION1, jobId3,
+        Status.RUNNING, null);
+
+    // Restarting table master will lose history for finished jobs,
+    // but history for running jobs are journaled and replayed.
+    restart();
+    assertEquals(1, mTableMaster.getAllTransformJobInfo().size());
+    checkTransformJobInfo(mTableMaster.getTransformJobInfo(jobId3), TABLE3, DEFINITION1, jobId3,
```
Comment (gpang) [inline table/server/master/src/test/java/alluxio/master/table/transform/TransformManagerTest.java]: after the restart, can we check the layout for job1 is the transformed layout?

Comment (chcn) [reply table/server/master/src/test/java/alluxio/master/table/transform/TransformManagerTest.java]: added

**Comment by chcn** [comment]
@gpang PTAL

**Comment by yuzhu** [comment]
jenkins, test this please

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by gpang** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please

**Pull request merged**